build/src/version.d.ts

```diff
@@ -0,0 +1,2 @@
+export declare const VERSION = "0.21.0";
```

Comment:
what is this version for?

Reply:
it was used in the otel collector-exporter, In our case it's redundant. I removed it.